backend/composer.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "ecamp/ecamp3",
     "description": "eCamp3",
-    "type": "library",
```

Comment:
das soll aber nicht in den devel
![image](https://user-images.githubusercontent.com/1506818/106387024-161cbd00-63d8-11eb-98c6-ee5de812b97e.png)
Für rebuild gibt es bei den checks oben rechts: Re run all all jobs

Reply:
Bin nicht ganz sicher was du meinst, "type": "project" scheint mir besser zu diesem Projekt zu passen als "library"..? Diskutieren könnte man höchstens ob das in diesen PR gehört. Aber da es so eine kleine Änderung ohne Auswirkungen ist ist mir persönlich das jetzt egal.

Reply:
Ich bin nur wegen der commit message für diesen change erschrocken,
und dass der change dann im PR drin geblieben ist

Reply:
@BacLuc das habe ich mehrere Male versucht gehabt. Es gab jedesmal einen Fehler mit der xdebug installation auf dem CI, obschon, dass ich dort nichts geändert habe. Durch mein Commit hat es den (Cache?) Fehler behoben.